### PR TITLE
Use font for all indian languages characters, not only Hindi in aww_n…

### PR DESCRIPTION
**Pull request by dmydlo**
…ame in pdf

Hi, @calellowitz,
Here I add font on Open Font Licence to be used for displaying aww name in CAS monthly register report that is exported as pdf.
I have added `link_callback` function that comes from xhtml2pdf documentation as xhtml2pdf requires absolute system paths to static files.

Regards, Dawid

### CODE REVIEW

File: custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html [hunk 1]
```diff
@@ -58,7 +63,7 @@ <h2>1. Basic Details</h2>
                                     <td>*Name Of Supervisor: </td>
                                 </tr>
                                 <tr>
-                                    <td>Name of AWW: {{ report.agg_awc_monthly_data|icds_format:"aww_name" }}</td>
+                                    <td>Name of AWW: <span style="font-family: HindGuntur-Regular">{{ report.agg_awc_monthly_data|icds_format:"aww_name" }}</span></td>
```
Comment (calellowitz) [inline custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html]: Does this affect outputs in latin characters as well?

**Review by calellowitz — APPROVED** [review]
one question but looks good

File: custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html (hunk 1 above)
Comment (dmydlo) [reply custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html]: Yes, yet this font has Latin characters as well in it.

**Comment by dmydlo** [comment]
Test failure not related (or even a failure of a test).

**Review by calellowitz — APPROVED** [review]

**Comment by calellowitz** [comment]
The current test failure (a 403 in the js tests) is quite consistent. Is it possible this is causing that in some way

**Comment by calellowitz** [comment]
@dmydlo 
That failure is not affecting other branches, just this one, and does so consistently. Are you able to reproduce the test failure locally?

**Comment by dmydlo** [comment]
@calellowitz no I am not able to reproduce it, this is RIAKCS connection error. As none of the existing libraries was altered I don't see the reason it would break right now. I will check this `Requirements are inconsistent.  Did you run 'make requirements'?` message

File: custom/icds_reports/utils/__init__.py [hunk 2]
```diff
@@ -605,6 +606,34 @@ def create_excel_file(excel_data, data_type, file_format):
     return file_hash
 
 
+def link_callback(uri, rel):
```
Comment (calellowitz) [inline custom/icds_reports/utils/__init__.py]: is this still necessary?

Comment (calellowitz) [reply custom/icds_reports/utils/__init__.py]: bump

Comment (dmydlo) [reply custom/icds_reports/utils/__init__.py]: No, it is no longer needed, I have removed it.

**Comment by dmydlo** [comment]
Hi @calellowitz,
I had merged the requirements.
Regards, Dawid

File: requirements/requirements.in [hunk 3]
```diff
@@ -105,6 +105,7 @@ csv342==1.0.0
 Faker==0.8.15
 polib==1.1.0
 werkzeug==0.11.15
+WeasyPrint==0.39
```
Comment (emord) [inline requirements/requirements.in]: Why is this using version 0.39? Quickly going over weasyprint's change log, it should be possible to use 0.42.3 with python 2.

Comment (dmydlo) [reply requirements/requirements.in]: You are right, I had to misinterpret the support part. I have tested it, 0.42.3 works as well, I am adding the change.

File: custom/icds_reports/utils/__init__.py [hunk 4]
```diff
@@ -16,10 +16,11 @@
 from base64 import b64encode
 from io import BytesIO
 from dateutil.relativedelta import relativedelta
+from django.conf import settings
 from django.template.loader import render_to_string, get_template
 from openpyxl.styles import PatternFill, Border, Side, Alignment, Font
 from openpyxl import Workbook
-from xhtml2pdf import pisa
```
Comment (emord) [inline custom/icds_reports/utils/__init__.py]: does this mean `xhtml2pdf` is no longer used? Can you remove that from requirements.in and add it to uninstall-requirements.txt

Comment (dmydlo) [reply custom/icds_reports/utils/__init__.py]: Yes, it was the last place where we used it. I have removed it from the requirements.

**Comment by dmydlo** [comment]
Hi @calellowitz, error in the tests is random and not related to the changes.

File: requirements/requirements.in (hunk 3 above)
Comment (calellowitz) [reply requirements/requirements.in]: I think you still need to add it to `uninstall-requirements.txt` as emord said

File: custom/icds_reports/utils/__init__.py (hunk 4 above)
Comment (calellowitz) [reply custom/icds_reports/utils/__init__.py]: I think you still need to add it to `uninstall-requirements.txt` as emord said

Comment (dmydlo) [reply custom/icds_reports/utils/__init__.py]: I have updated that and fixed conflicts.

**Pull request merged**